app/src/main/java/com/xinto/opencord/ui/components/channel/ChatInput.kt

```diff
@@ -41,13 +43,15 @@ fun ChatInput(
                 CompositionLocalProvider(LocalAbsoluteTonalElevation provides 0.dp) {
                     Surface(
                         modifier = Modifier.weight(1f),
-                        shape = MaterialTheme.shapes.medium,
+                        shape = RoundedCornerShape(50.dp),
                     ) {
-                        Box(modifier = Modifier.padding(12.dp)) {
+                        Box(modifier = Modifier.padding(14.dp)) {
                             innerTextField()
                             CompositionLocalProvider(
                                 LocalContentAlpha provides ContentAlpha.medium,
-                                LocalTextStyle provides MaterialTheme.typography.bodyMedium,
+                                LocalTextStyle provides MaterialTheme.typography.titleMedium.copy(
+                                    fontWeight = FontWeight.Medium
```

Comment:
this also looks weird
too thick

heres my stock for reference
![image](https://user-images.githubusercontent.com/33725716/216446043-3809065d-4f21-46f1-a71d-57ff60c39d3c.png)